packages/experimental/kai-framework/src/hooks/useClientProvider.ts

```diff
@@ -18,7 +18,7 @@ export const useClientProvider = (dev: boolean) => {
     const config = await configProvider();
     const client = new Client({
       config,
-      services: config.get('runtime.app.env.DX_VAULT') === 'false' ? fromHost(config) : fromIFrame(config),
+      services: config.get('runtime.app.env.DX_VAULT') === 'false' ? fromHost({ config }) : fromIFrame(config),
```

Comment:
Perhaps factor out (or look for) a parseBool that treats "0" as false also. I think we do this a lot.